src/GitReportCL/GitReportErrorsHandler.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+namespace GitReport.CLI
+{
+    class GitReportErrorsHandler
+    {
+        public void HandleDateFormatOrPathError(string[] currentArguments, 
+            GitReportArguments getArg, out string[] newArguments)
+        {
+            if (getArg.DateSince == DateTime.MinValue)
+            {
+                currentArguments[0] = EnterDate("start-date", true);
+            }
+
+            if (getArg.DateBefore == DateTime.MinValue)
+            {
+                currentArguments[1] = EnterDate("end-date", true);
+            }
+
+            if (getArg.DateSince > getArg.DateBefore &&
+                getArg.DateSince != DateTime.MinValue &&
+                getArg.DateBefore != DateTime.MinValue)
+            {
+                Console.WriteLine("The end-date was more previous then start - date.");
+                currentArguments[0] = EnterDate("start-date", false);
+                currentArguments[1] = EnterDate("end-date", false);
+            }
+
+            if (getArg.GitPath == "no path found")
```

Comment:
in this value you disable user having code in `no path found` folder, application should not do this. Application should allow user to have phoenix source code in every foldername possible